src/app/components/Dropdown.tsx

```diff
@@ -58,9 +58,12 @@ const TabHiddenDropDown = styled(({toggle, children, className}: Props) => {
   const container = React.useRef<HTMLElement>(null);
 
   useOnClickOutside(container, open, () => setOpen(false));
-  useOnEsc(container, open, () => setOpen(false));
+  useOnEsc(container, open, () => {
+    setOpen(false);
+    if (container.current) { container.current.focus(); }
+  });
 
-  return <div className={className} ref={container}>
+  return <div className={className} ref={container} tabIndex={0}>
```

Comment:
isn't this going to clutter up the keyboard navigation? maybe put it to `-1`? i also think it would be good to focus the toggle, instead of the wrapper, unless you had some problem doing that

Reply:
Not sure how to pass `ref` to `toggle` component since we are using `React.cloneElement` wrapped with styled components and it would have to be somehow wrapped with `React.forwardRef`